src/components/AuthorCardEditing.vue

```diff
@@ -94,6 +94,7 @@
                 bg-color="white"
                 class="col"
                 dense
+                hint="https://orcid.org/0000-0000-0000-0000"
```

Comment:
The error message replaces the hint. Which makes initial visit better, but once there is an error it is not useful anymore.